entgql/extension.go

```diff
@@ -136,12 +141,57 @@ func WithTemplates(templates ...*gen.Template) ExtensionOption {
 	}
 }
 
+// WithNaming takes either "snake" or "camel" and sets that as the naming
+// convention when generating the GraphQL schema
+func WithNaming(s string) ExtensionOption {
```

Comment:
This is quite odd. I have encountered a snake_case graphql schema. Is this a use case you specifiically need?

Reply:
In our case, the schema is something that the users will interact with directly, in various different ways and so far it is `snake_case` everywhere... and whilst this is not "standard" for Graphql I think it would be nice to support.
So yes, it's a case we would really like